### PR TITLE
feat(ui): Beautify the Service Incidents flyout

### PR DESCRIPTION
**Pull request by evanpurkhiser**
It looked like this before

image

now it looks like this

image

### CODE REVIEW

File: static/app/actionCreators/serviceIncidents.tsx [hunk 1]
```diff
@@ -9,9 +9,11 @@ type IncidentImpact = SentryServiceStatus['indicator'];
  * [0]: https://doers.statuspage.io/api/v2/incidents/
  */
 type StatuspageIncident = {
+  components: Array<any>;
+  created_at: string;
   id: string;
   impact: IncidentImpact;
-  incident_updates: {body: string}[];
+  incident_updates: Array<any>;
```
Comment (evanpurkhiser) [inline static/app/actionCreators/serviceIncidents.tsx:16]: 🤷 Thes types are already totally incomplete anyway

**Review by priscilawebdev — COMMENTED** [review]
it looks much nicer now 🎉

should we write tests for these changes?

**Comment by evanpurkhiser** [comment]
> should we write tests for these changes?

it would be cool to get a visual snapshot of it I think actually

**Comment by priscilawebdev** [comment]
What does it look like in small viewports? @evanpurkhiser

**Comment by evanpurkhiser** [comment]
> What does it look like in small viewports

Takes the full screen. It looks good I proimse!

**Review by priscilawebdev — APPROVED** [review]
I haven't tested it locally, but the code looks good. It would be nice to have visual and unit tests too to avoid future regressions 😉

**Comment by evanpurkhiser** [comment]
Spending too much time trying to figure out why the sidebar flyout isn't rendering in snapshots.

It's related to the fact that the sidebar is rendered in a portal.. but going to stop debugging for now and just merge this.

**Pull request merged**